style.css

```diff
@@ -2189,6 +2189,7 @@ body.search-no-results .page-header {
 	 * TODO custom thumbnail size for type / crop in UI on upload. */
 	height: 300px;
 	overflow: hidden;
+	margin-top: -0.5em;
```

Comment:
Negative margins are not so pretty, rather use larger bottom margin on element before it.

Reply:
adressed see new commit